### ISSUE TITLE
Supervisor hang until job killed

### ISSUE THREAD

**Issue by robnagler** [radiasoft/sirepo#4068]
The hang occurred at 17:38:54 and the server continued at 17:39:36:

```txt
17:39:36 sirepo/job_supervisor.py:984:run_timeout _Op(analysis, 0FhG) max_run_secs=144
```

The op started at 17:37:07

```
17:37:07 sirepo/job_supervisor.py:937:__init__ _Op(analysis, 0FhG) runDir=None
``

There was a curious exception at 17:38:32:

```txt
17:38:32 sirepo/job_supervisor.py:1005:set_job_situation _Op(analysis, cmha) situation=Waiting for another simulation to complete stack=Traceback (most recent call last):
17:38:32 sirepo/job_supervisor.py", line 1002, in set_job_situation
17:38:32 yield
17:38:32 sirepo/job_supervisor.py", line 111, in alloc
17:38:32 self._value = await self._q.get()
17:38:32 sirepo/tornado.py", line 49, in get
17:38:32 return await x
```

There is also a strange sequence that's probably unrelated regarding restarts of the agent, but his one might be relevant:

```txt
17:37:07 sirepo/job_driver/__init__.py:263:_agent_start DockerDriver(a=qtZ9 k=sequential u=uid0 [_Op(analysis, 0FhG)]) _Op(analysis, 0FhG) await _do_agent_start
```

There are numerous other starts/stops, but it's the same user so unclear why the agent is being killed and restarted. The server was lightly loaded.

See [full log](https://github.com/radiasoft/sirepo/files/7916585/20220121-log.txt).

**Comment by robnagler** [comment]
@moellep do you know what the statefulCompute was that was running when it was hung? We don't have enough logging to tell us.

**Comment by moellep** [comment]
I'm not sure which one was running - I was opening a SRW sim so many reports may have started, and I changed to the beamline page quickly. Possibly the sourceIntenityReport. If we add more logging, I'm sure this problem will occur later to debug.

**Comment by robnagler** [comment]
It hit the agent. We see that in the log, but there's no "reply", because we don't log that (see #4086). I was wondering if it could possibly have run for 144 seconds, but I guess not.

**Comment by moellep** [comment]
I don't think it should have run for 144 seconds - it was one of the example simulations which should run quickly.
